app/brave_generated_resources.grd

```diff
@@ -213,6 +213,7 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       </message>
     </messages>
     <includes>
+      <include name="IDR_BRAVE_NICEWARE_LIST" file="../vendor/bat-native-ledger/niceware/wordlist" type="BINDATA" />
```

Comment:
I think this would make more sense in the existing brave_rewards/resources.grd or at least brave_components_resources.grd if there are any base_dir issues in brave_rewards/resources.grd
cc @bbondy

Reply:
I'm going to push it to brave_components_resources.grd since the brave_rewards resources have been pushed under extensions.

Reply:
Moved.